plugins/schedule/base/lib/model/ScheduleEvent.php

```diff
@@ -162,6 +178,106 @@ public function getRecurrence()
 		return $this->getFromCustomData(self::CUSTOM_DATA_FIELD_RECURRENCE);
 	}
 	
+	public function getLinkedTo()
+	{
+		return $this->getFromCustomData(self::CUSTOM_DATA_FIELD_LINKED_TO);
+	}
+	
+	public function getLinkedBy()
+	{
+		return $this->getFromCustomData(self::CUSTOM_DATA_FIELD_LINKED_BY);
+	}
+	
+	public function setLinkedTo($v)
+	{
+		$this->putInCustomData(self::CUSTOM_DATA_FIELD_LINKED_TO, $v);
+	}
+	
+	public function setLinkedByString($v)
+	{
+		$this->putInCustomData(self::CUSTOM_DATA_FIELD_LINKED_BY, $v);
+	}
+	
+	public function addAnotherLinkedBy($v)
+	{
+		$linkedByString = $this->getLinkedBy();
+		if (strpos($linkedByString, strval($v)) === false)
+		{
+			$linkedByString = $linkedByString . $v . ',';
+			$this->removeFromCustomData(self::CUSTOM_DATA_FIELD_LINKED_BY);

```

Comment:
remove is not needed - put will override it.
Remove is needed if we want to delete the field

Reply:
done